test/e2e/scheduling/equivalence_cache_predicates.go

```diff
@@ -35,6 +35,10 @@ import (
 	_ "github.com/stretchr/testify/assert"
 )
 
+const (
+	defaultTimeout = 3 * time.Minute
```

Comment:
Why is this needed? It does not seem to belong to this PR.

Reply:
The `defaultTimeout` from rescheduler.go(which I removed) is being used in equivalence_cache_predicates.go :(.  So, had to add it here..

>  Please make sure that the test failures are not caused by this PR.

Thanks for the review. Looks like they are not related. I will rebase the PR later in the evening and will ask the bot to test the changes again.